docker-compose.yml

```diff
@@ -0,0 +1,31 @@
+services:
+  cinema:
+    build:
+      context: .
+    env_file:
+      - .env
+    ports:
+      - "8000:8000"
+    volumes:
+      - ./:/app
```

Comment:
Mounting the entire current directory into the container with `- ./:/app` can lead to potential issues with file permissions and unnecessary files being included. Consider using a more specific path or ensuring your `.dockerignore` file is properly configured to exclude unnecessary files.